pulumi/python/kubernetes/observability/otel-operator/opentelemetry-operator.yaml

```diff
@@ -289,6 +421,7 @@ spec:
                               required:
                               - key
                               type: object
+                              x-kubernetes-map-type: atomic
```

Comment:
For these things, do we want to add some link or comment to help users understand what this is?  I found this https://kubernetes.io/docs/reference/using-api/_print/#merge-strategy

Reply:
Same as above...